CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Unreleased:
 - Added a password encoder to the client entity service
 - Fixes a bug by specifying the name of the scope columnn
 - Removed functionality that passed the client secret down to the UI
+- Updated JDK to Corretto 1.8.252
```

Comment:
I'm not sure there's much worth maintaining the change log here given we don't have anything in other products.

Reply:
We've been maintaining a changelog in some of our newer and smaller repos and it's been quite useful.
I don't plan to make any further changes in this repo!